programs/uxd/src/state/mercurial_vault_depository.rs

```diff
@@ -4,25 +4,29 @@ use fixed::types::I80F48;
 use crate::error::UxdError;
 
 // Total should be 900 bytes
-pub const MERCURIAL_VAULT_RESERVED_SPACE: usize = 645;
+pub const MERCURIAL_VAULT_RESERVED_SPACE: usize = 588;
 pub const MERCURIAL_VAULT_DEPOSITORY_SPACE: usize = 8
-    + 1
-    + 1
-    + 32
-    + 1
-    + 32
-    + 16
-    + 16
-    + 32
-    + 32
-    + 1
-    + 32
-    + 1
-    + 1
-    + 1
-    + 16
-    + 16
-    + 16
+    + 1     // bump
+    + 1     // version
+    + 32    // collateral mint
+    + 1     // collateral mint decimals
+    + 32    // controller
+    + 16    // collateral amount deposited
+    + 16    // redeemable amount under management
+    + 32    // mercurial vault
+    + 32    // mercurial vault lp mint
+    + 1     // mercurial vault lp mint decimals
+    + 32    // lp token vault
```

Comment:
micro-nits:

- could use the notation: size_of::<Pubkey>()
- could use the variable name (with the underscores) for easy grep